LiveSplit.AutoSplitters.xml

```diff
@@ -1710,6 +1710,17 @@
         </URLs>
         <Type>Component</Type>
         <Description>Auto Splitting is available. (By ShootMe)</Description>
+    </AutoSplitter>
+		<AutoSplitter>
+        <Games>
+            <Game>Dishonored Category Extensions</Game>
+            <Game>Dishonored (PC)</Game>
+        </Games>
+        <URLs>
+            <URL>http://fatalis.pw/livesplit/update/Components/LiveSplit.Dishonored.dll</URL>
+        </URLs>
+        <Type>Component</Type>
+        <Description>Load Removal and Auto Splitting are available. (By Fatalis)</Description>
     </AutoSplitter>
     <AutoSplitter>
         <Games>
```

Comment:
You should add the Category Extensions entry to the existing autosplitter to avoid duplication and unnecessary/misleading future changes for the Dishonored splitter.
```suggestion
```